opcua/common/xmlexporter.py

```diff
@@ -310,19 +270,20 @@ def _add_refs(self, obj):
         #     refs.append(ref)
         # self.server.add_references(refs)
 
-    @staticmethod
-    def _get_xml_nodeid(obj):
-        """
-        Convert a UA NodeId object to a formatted string for XML
-        :param obj:
-        :return:
-        """
-        return 'ns=' + str(obj.nodeid.NamespaceIndex) + ';' + str(obj.nodeid.Identifier)
+    def _get_xml_namespace_uris(self):
+        # TODO name space uris should be exported
+        pass
+
+    def _get_xml_aliases(self):
+        # TODO aliases need to be created at the top of the xml
+        pass
 
     @staticmethod
-    def _get_xml_browsename(obj):
-        bn = obj.get_browse_name()
-        return str(bn.NamespaceIndex) + ':' + bn.Name
+    def _add_ref_sub_els(parent_el, refs):
+        refs_el = Et.SubElement(parent_el, 'References')
 
-    def _get_xml_parent(self, obj):
-        return self._get_xml_nodeid(obj.get_parent())
\ No newline at end of file
+        for ref in refs:
+            ref_name = o_ids.ObjectIdNames[ref.ReferenceTypeId.Identifier]
+            ref_forward = str(ref.IsForward)
+            refx_el = Et.SubElement(refs_el, 'Reference', IsForward=ref_forward, ReferenceType=ref_name)
```

Comment:
When I look at UAExpert exports it doesn't always include "IsForward". How do I know if this is required or not?